modules/ROOT/pages/_partials/nav-dw.adoc

```diff
@@ -1,4 +1,4 @@
-** xref:dw-core.adoc[Core (dw::Core)]
+** xref:dw-core.adoc[dw::Core]
```

Comment:
NOTE: manually fixed H level to one level higher)
TODO: create DW Jira for builder to change the output.